src/gui/dialogues/newpilotdialog.h

```diff
@@ -38,19 +38,15 @@ class NewPilotDialog : public QDialog
 {
     Q_OBJECT
 public:
-    explicit NewPilotDialog(Db::editRole, QWidget *parent = nullptr);
-    explicit NewPilotDialog(Pilot, Db::editRole, QWidget *parent = nullptr);
-    explicit NewPilotDialog(experimental::PilotEntry oldEntry, Db::editRole, QWidget *parent = nullptr);
+    explicit NewPilotDialog(QWidget *parent = nullptr);
+    explicit NewPilotDialog(int rowId, QWidget *parent = nullptr);
```

Comment:
aaaah yeshhh thats what i like to see. Simple cleaner constructors. Almost better than ice cream... almost...